content/oddnews.njk

```diff
@@ -21,12 +24,13 @@ summary: |
 {%- set title = 'What to Expect' -%}
 {%- set contents -%}
   Here are a few examples of topics featured in past editions of OddNews.
-  - **Bird of the Month** - Introduction to the oddest of birds
   - **CSS** - [Miriam Suzanne's](/authors/miriam/) latest work in Container Queries
-  - **Python** - How to Run Playwright Inside Docker
   - **Sass** - How to use the new `split()` function in Sass
```

Comment:
I wonder if we should drop the word "new" on a few of these so it is a bit more long-term if we don't want to update them frequently.